pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
```

Comment:
We already have this file, please remove.
Ours is located in here
https://github.com/pymasterspl/Dshop/blob/dev/Dshop/pyproject.toml

Reply:
fixed :)